core/src/main/java/org/apache/calcite/config/CalciteConnectionConfigImpl.java

```diff
@@ -199,4 +199,8 @@ public boolean lenientOperatorLookup() {
     return CalciteConnectionProperty.LENIENT_OPERATOR_LOOKUP.wrap(properties)
         .getBoolean();
   }
+
+  public boolean topDownOpt() {
```

Comment:
Can you add a comment that this value will override the value from `saffron.properties`?

Reply:
This parameter actually does not overwrite `saffron.properties`. The comment below.